### PR TITLE
add acrn-kernel 5.4 recipe

### PR DESCRIPTION
**Pull request by wenlingz**
Add acrn-kernel 5.4 recipe
Signed-off-by: wenlingz <wenling.zhang@intel.com>

### CODE REVIEW

**Comment by wenlingz** [comment]
@saininav  could you help review and merge this PR, thanks

File: conf/distro/acrn-demo-sos.conf [hunk 1]
```diff
@@ -5,6 +5,7 @@ DISTRO_NAME += "(SOS)"
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-intel-acrn-sos"
 PREFERRED_VERSION_linux-intel-acrn-sos ?= "5.4%"
+PREFERRED_VERSION_acrn-kernel-sos ?= "5.4%"
```
Comment (saininav) [inline conf/distro/acrn-demo-sos.conf:8]: This change is not required.

File: conf/distro/acrn-demo-uos.conf [hunk 2]
```diff
@@ -6,6 +6,7 @@ DISTRO_NAME += "(UOS)"
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-intel-acrn-uos"
 PREFERRED_VERSION_linux-intel-acrn-uos ?= "5.4%"
 PREFERRED_VERSION_linux-intel-rt-acrn-uos ?= "5.4%"
+PREFERRED_VERSION_acrn-kernel-uos ?= "5.4%"
```
Comment (saininav) [inline conf/distro/acrn-demo-uos.conf:9]: Drop this change too..

File: conf/distro/acrn-demo-sos.conf (hunk 1 above)
Comment (wenlingz) [reply conf/distro/acrn-demo-sos.conf]: done

**Pull request merged**